WordPress/UITestsFoundation/Screens/ReaderScreen.swift

```diff
@@ -94,9 +119,36 @@ public class ReaderScreen: ScreenObject {
         (try? ReaderScreen().isLoaded) ?? false
     }
 
-    public func openDiscover() -> ReaderScreen {
+    public func openDiscover() -> Self {
         discoverButton.tap()
 
         return self
     }
+
+    public func selectTopic() -> Self {
+        topicCellButton.firstMatch.tap()
+
+        return self
+    }
+
+    public func verifyTopicLoaded() -> Self {
+        XCTAssertTrue(topicNavigationBar.waitForExistence(timeout: 3))
+        XCTAssertTrue(readerButton.waitForExistence(timeout: 3))
+        XCTAssertTrue(followButton.waitForExistence(timeout: 3))
+
+        return self
+    }
+
+    public func followTopic() -> Self {
+        followButton.tap()
+
+        return self
+    }
+
+    @discardableResult
+    public func verifyTopicFollowed() -> Self {
+        XCTAssertTrue(followingButton.waitForExistence(timeout: 3) && followingButton.isSelected)
```

Comment:
Total nitpick: IMO, splitting this into two assertions will make it easier to know which one of the two has failed 🤔

Reply:
yup that's reasonable because the other new assert method has separate assertations (though they are different elements), change made in c3f7b6c83eee474422f83c572ffff427dfa8fcad

thanks for the review!